setup.cfg

```
@@ -67,6 +67,7 @@ optional =
     pyshp
     python-dateutil >=2.4.0
     rasterio; platform_system!='Windows'
+	rasterstats; platform_system!='Windows'
```

minor nit to change 1 <kbd>tab</kbd> -> 4 spaces
```suggestion
    rasterstats; platform_system!='Windows'
```
On a related note, rasterio [1.3.2](https://pypi.org/project/rasterio/1.3.2/#files) has binary wheels for windows for py38+, and fiona's [1.9a2 pre-release](https://pypi.org/project/Fiona/1.9a2/#files) has binary wheels for py37+, which means that `pip install rasterstats --pre` works on windows with py38+. I'll update this when these external releases are ready and the minimum Python version is bumped to py38, which I'll do after [3.3.6](https://github.com/modflowpy/flopy/milestone/10) is done.